.github/workflows/ci.yml

```diff
@@ -53,6 +53,25 @@ jobs:
 
       - run: yarn test
 
+  ember-template-lint:
+    name: ember-template-lint@${{ matrix.etlVersion }}
+    needs: [test]
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    strategy:
+      matrix:
+        etlVersion: [^4.0.0, 5.0.0-alpha.0]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: volta-cli/action@v1
```

Comment:
```suggestion
      - uses: actions/checkout@v3
      - uses: volta-cli/action@v4
```

Reply:
This just uses the same versions as the other jobs, I could make a separate PR to bump everything and then rebase this one?

Reply:
Made #238.

Reply:
Yep, separate PR is fine.